packages/components/FilePreview/SelectedFilesPreview.tsx

```diff
@@ -0,0 +1,188 @@
+                      }}
+                    >
+                      <img
+                        src={URL.createObjectURL(item.file)}
+                        style={{
+                          height: 98,
+                          width: 98,
+                          borderRadius: 8,
+                        }}
+                        alt="Uploaded file"
+                      />
+                    </PrimaryBox>
+
+                    <PrimaryBox
+                      style={{
+                        borderRadius: 32,
+                        height: 34,
+                        width: 52,
+                        top: 0,
+                        alignItems: "center",
+                        justifyContent: "center",
+                        position: "absolute",
+                      }}
+                    >
+                      <BrandText
+                        style={[fontSemibold13, { color: secondaryColor }]}
+                      >
+                        {currentPage * itemsPerPage + index + 1}
+                      </BrandText>
+                    </PrimaryBox>
+                  </TouchableOpacity>
+                ))}
+              </View>
+              <View style={{ flexDirection: "row" }}>
+                {currentItems.slice(7).map((item, index) => (
+                  <TouchableOpacity
+                    style={{
+                      height: 123,
+                      width: 100,
+                      marginTop: layout.spacing_x2,
+                      justifyContent: "flex-end",
+                      alignItems: "center",
+                      marginHorizontal: layout.spacing_x1,
+                    }}
+                    onPress={() => {
+                      onSelect(item);
+                    }}
+                  >
+                    <PrimaryBox
+                      style={{
+                        height: 100,
+                        width: 100,
+                      }}
+                    >
+                      <img
+                        src={URL.createObjectURL(item.file)}
+                        style={{
+                          height: 98,
+                          width: 98,
+                          borderRadius: 8,
+                        }}
+                        alt="Uploaded file"
+                      />
+                    </PrimaryBox>
+
+                    <PrimaryBox
+                      style={{
+                        borderRadius: 32,
+                        height: 34,
+                        width: 52,
+                        top: 0,
+                        alignItems: "center",
+                        justifyContent: "center",
+                        position: "absolute",
+                      }}
+                    >
+                      <BrandText
+                        style={[fontSemibold13, { color: secondaryColor }]}
+                      >
+                        {currentPage * itemsPerPage + index + 1 + 7}
```

Comment:
if 7 is not "test" code then could you please create a constant with a explicative name ?

Reply:
Okay, I will fix it.